examples/free_convection.jl

```diff
@@ -0,0 +1,60 @@
+using 
+    Plots, 
+    PyPlot, 
+    FFTW, 
+    Oceananigans
+
+function make_temperature_movie(model::Model, fw::NetCDFOutputWriter)
+    n_frames = Int(model.clock.time_step / fw.frequency)
+
+    xC, yC, zC = model.grid.xC, model.grid.yC, model.grid.zC
+
+    print("Creating temperature movie... ($n_frames frames)\n")
+
+    Plots.gr()
+    default(dpi=300)
+    movie = @animate for tidx in 0:n_frames
+        print("\rframe = $tidx / $n_frames   ")
+        temperature = read_output(fw, "T", tidx*fw.frequency)
+        Plots.heatmap(xC, zC, rotl90(temperature[:, Int(ceil(model.grid.Ny/2)), :]) .- 283, color=:balance,
+                      clims=(-0.5, 0.5), aspect_ratio=:equal,
+                      title="T @ t=$(tidx*fw.frequency*model.clock.Δt)")

```

Comment:
Should now be
```julia
title="T @ t=$(tidx*fw.frequency)")
```
to agree with line 18 as files are named after the time step iteration number.

Reply:
I thought I made that change, but maybe not?